src/values.js

```diff
@@ -160,6 +160,54 @@ export class FloatValue extends Value {
   }
 }
 
-export {
-  process,
-};
+export class DurationValue extends IntegerValue {
+  static SECOND = 1000;
+  static MINUTE = 60 * DurationValue.SECOND;
+  static HOUR = 60 * DurationValue.MINUTE;
+  static DAY = 24 * DurationValue.HOUR;
+
+  toJS(value) {
+    let quantity, unit;
```

Comment:
Perhaps add...
```javascript
if (typeof value === 'number') {
  return value
}
```

Reply:
oh wait. there's already a unit test that tests setting the default to be a number. Then I'm not sure I get how this works.

Reply:
`DurationValue` extends another class `IntegerValue` that has the validation.

Reply:
Oh also default values are not passed through `.toJS()`.

They are however validated using `.validateDefaultsTo()`. In this case: https://github.com/rehandalal/node-env-settings/blob/master/src/values.js#L115

That function simply throws errors if the value does not validate but doesn't do conversion so the default value must be an integer.